scala-thoughts.md

```diff
@@ -0,0 +1,45 @@
+# What's easy to do in Scala? What's not?
+It's really easy to do method chaining in Scala. When I needed map, then filter called on only part of a list
+I could make all of those method calls on one line and Scala was very okay with it. Additionally, objects 
+are very easy to implement in Scala because everything has to be a class already, so making the class some sort
+of intersting object is not difficult. With that, inheritence seems to be pretty easy. We only had to use Any for
+this assignment. However, from reading the documentation on Sequences and other Collections, it seemed that 
+inheritence was very common and calling super methods was simple.
+
+In Scala, it is difficult to learn collection syntax. Although it is simple to use them functionally, with map and
+filter, it was very difficult for me to learn the difference between `::`, `+:`, and `++` and some other minor syntax
+issues with collections. There were a couple other obscure syntax issues, especially with sumOfMany, which used the *
+operator to note that there were many lists. It proved very difficult to find documentation or help on some more
+obscure issues of Scala.

```

Comment:
I found this to be somewhat of a problem too. When I knew what the thing I was looking for was called, the documentation was easy to find and helpful, but trying to find info about things I didn't know the name of was rather difficult.